packages/core/src/components/Drawer/Drawer.tsx

```diff
@@ -81,41 +88,61 @@ export const HvDrawer = (props: HvDrawerProps) => {
     onClose,
     anchor = "right",
     buttonTitle = "Close",
+    disableBackdropClick = false,
     ...others
   } = useDefaultProps("HvDrawer", props);
 
-  const { classes, cx } = useClasses(classesProp);
+  const { classes, cx, css } = useClasses(classesProp);
+  const { colors } = useTheme();
 
   const closeButtonDisplay = () => <Close role="none" />;
 
   const CloseButtonTooltipWrapper = buttonTitle
     ? withTooltip(closeButtonDisplay, buttonTitle, "top")
     : closeButtonDisplay;
 
+  const handleBackdropClick = useCallback(
+    (event) => {
```

Comment:
Can we type the function? `event` is any like this. Alternatively, we could inline the function 😄 
The memo seems unnecessary too.